src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.cxx

```diff
@@ -1019,6 +1031,7 @@ PlusStatus vtkPlusBkProFocusOemVideoSource::ReadConfiguration(vtkXMLDataElement*
   XML_READ_SCALAR_ATTRIBUTE_REQUIRED(int, OemPort, deviceConfig);
   XML_READ_BOOL_ATTRIBUTE_OPTIONAL(ContinuousStreamingEnabled, deviceConfig);
   XML_READ_BOOL_ATTRIBUTE_OPTIONAL(ColorEnabled, deviceConfig);
+  XML_READ_SCALAR_ATTRIBUTE_OPTIONAL(double, AcquisitionRate, deviceConfig);

```

Comment:
I believe this is handled by the base class ReadConfiguration?

Reply:
Yes, and the base class read configuration is handled by the `XML_FIND_DEVICE_ELEMENT_REQUIRED_FOR_READING` macro. This isn't needed.